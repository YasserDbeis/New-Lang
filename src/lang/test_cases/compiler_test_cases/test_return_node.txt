func main(int x) -> int
{
    return 5;
    return 5;
    return 5;
    return 5;
    return 5;
    return 5;
    return 5;
    return 5;

    func_call();
}