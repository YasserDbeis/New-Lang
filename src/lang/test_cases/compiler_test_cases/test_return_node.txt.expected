NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE
NODE ID: RETURN  NODE